.pre-commit-config.yaml

```diff
@@ -1448,7 +1448,14 @@ repos:
         entry: ./scripts/ci/pre_commit/update_fastapi_api_spec.py
         pass_filenames: false
         files: ^airflow/api_fastapi/.*\.py$
+        exclude: ^airflow/api_fastapi/execution_api/.*
         additional_dependencies: ['rich>=12.4.4']
+      - id: generate-tasksdk-datamodels
+        name: Generate Datamodels for TaskSDK client
+        language: system
+        entry: uv run --group codegen --project apache-airflow-task-sdk --directory task_sdk python dev/generate_models.py
+        pass_filenames: false
+        files: ^airflow/api_fastapi/execution_api/.*\.py$
```

Comment:
You should add `require_serial` as well - while this is pretty illogical to have `pass_filenames: false` and `require_serial: false`, I think pre-commit might launch multiple instances when you run `--all-files` if you do not explicitly set `require_serial`. It's better to add it explicitly.

Reply:
🤦🏻

Reply:
Yeah. Try to convinve the pre-commit author that it's not the right choice :) . Good luck.